libs/generic-view/feature/src/lib/recursive-layout/recursive-layout.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React, { FunctionComponent } from "react"
+import { useSelector } from "react-redux"
+import { ReduxRootState } from "Core/__deprecated__/renderer/store"
+import apiComponents from "generic-view/ui"
+import { APIFC, withLayout } from "generic-view/utils"
+
+interface MetaData {
```

Comment:
🟢 

```suggestion
interface Props {
```

Instead of the name `MetaData`, consider renaming the interface to `Props`, which more accurately reflects that the interface defines component properties.

I suggest this change to facilitate using this interface in other components and to avoid the need for creating special names.

Reply:
In this case, I think `MetaData` is more meaningful than the generic name `Props`. So I would suggest sticking to `MetaData` or considering something more unique like `RecursiveLayoutProps`

Reply:
Thanks for your perspective. I believe the property doesn't need to have a unique name until it's exported, and the idea of `RecursiveLayoutProps` works for me. I've marked my opinion with a green dot, so the author can decide what to do with it.

Reply:
This interface is used only by this component itself (no `export`), so it's name doesn't really matter. Just "Properties" should be fine.